test/module/irohad/consensus/yac/yac_synchronization_test.cpp

```diff
@@ -43,15 +43,12 @@ class NetworkUtil {
       const iroha::consensus::Round &r,
       const std::string &block_hash = "default_block",
       const std::string &proposal_hash = "default_proposal") const {
-    return std::accumulate(
-        peers.begin(),
-        peers.end(),
-        std::vector<VoteMessage>(),
-        [&, this](auto vector, const auto &peer_number) {
-          vector.push_back(this->createVote(
-              peer_number, this->createHash(r, block_hash, proposal_hash)));
-          return std::move(vector);
-        });
+    std::vector<VoteMessage> votes;
+    for (const auto &peer_number : peers) {
```

Comment:
We know peers.size(). votes.size() will be equal, so we can
votes.reserve(peer_number.size());